docs/manual/forms/examples/registration_attributes.rst

```diff
@@ -0,0 +1,95 @@
+.. _example_registration_attributes:
+
+=================================
+Form with registration attributes
+=================================
+
+When adding a component to a form, it is possible to have the value filled in by
+the user set as a specific field on the selected registration backend.
+
+In this example, we will use the `Objects API`_ as a registration backend.
+
+.. note::
+
+    Depending on the selected backend, the supported registration attributes might differ.
+    In the case of `Objects API`_, only the *Location > Coordinate* attribute is supported.
+
+
+Creating the form
+=================
+
+1. Create an empty form and add a first step.
+2. Add a new map component: *Speciale velden > Kaart*.
+3. Under the **Registratie** tab, select the value *Location > Coordinate* in the **Registratie-attribuut** dropdown.
+4. Save the component, and go under the **Registration** tab of the current form.
+5. Add one of your configured Objects API backends (see :ref:`configuration_registration_objects` for more details on how to configure the backend).
+
+Registration process
+====================
+
+When a new submission for this form is created, it will be registered in the selected backend (in our case, Objects API).
+
+Internally, Open Forms will look into each component to see if a registration attribute is set. If this is the case,
+the submitted value for this component will be set to a fixed JSON path (this path differs depending on the registration backend).
+
+In the case of `Objects API`_, the service supports a geometry attribute set to ``record.geometry``. In our case,
+the JSON data sent to the Objects API will look like the following:
+
+.. code-block:: json
+
+    {
+        "type": "<configured_objecttype>",
+        "record": {
+            "typeVersion": "<configured_version>",
+            "data": "<rendered_configured_json_template>",
+            "startAt": "YYYY-MM-DD",
+            "geometry": "<The submitted value of the map component, in GeoJSON>"
+        }
+    }
```

Comment:
Examples need to be concrete, don't use placeholders that the reader needs to fill out mentally. The target users for this documentation are not developers, but people building forms. "JSON" as a concept may already be too much.